control-plane/agents/src/bin/core/tests/pool/mod.rs

```diff
@@ -867,7 +867,13 @@ async fn disown_unused_replicas() {
     let volume = volumes_api
         .put_volume(
             &"1e3cf927-80c2-47a8-adf0-95c481bdd7b7".parse().unwrap(),
-            models::CreateVolumeBody::new(models::VolumePolicy::default(), 1, 5242880u64, false),
+            models::CreateVolumeBody::new(
```

Comment:
not for this PR;
we should modify the openapi generated code to use builder pattern
I had intended to eventually modify it to match the paperclip v2 generated code but never got around to that

Reply:
Yeah, that would be really nice actually, we wouldn't need to change all the code everywhere in tests. Just set the needed ones.